.github/workflows/ci.yml

```diff
@@ -112,6 +121,22 @@ jobs:
       - name: Test current JDBC vs old server
         if: always()
         run: $MAVEN test ${MAVEN_TEST} -pl :trino-test-jdbc-compatibility-old-server
+      - name: Upload test results
+        uses: actions/upload-artifact@v2
+        if: failure()
+        with:
+          name: result ${{ github.job }}
+          path: |
+            **/target/surefire-reports
+            **/target/checkstyle-*
+      - name: Upload test report
+        uses: actions/upload-artifact@v2
+        if: always()
+        with:
+          name: test report ${{ github.job }}
+          path: |
+            **/surefire-reports/TEST-*.xml
```

Comment:
Seems to duplicate `**/target/surefire-reports` above

Reply:
I added comments explaining this in every job.